### PR TITLE
markdown images support

### PR DESCRIPTION
**Pull request by davidbinyamin**
Related issue: https://github.com/demisto/etc/issues/40210

 - Support markdown resources server parameter (to be sent via sanePdfReports script)
 - translate any demisto markdown image source (hosted in demisto server) to the markdown resource server received via the cmd parameter.

## Related PRs
branch | PR
------ | ------
Content PR | https://github.com/demisto/content/pull/14508
Server MR |  https://code.pan.run/xsoar/server/-/merge_requests/16194

### CODE REVIEW

File: src/constants/Constants.js [hunk 1]
```diff
@@ -197,3 +198,6 @@ export const DATA_TYPES = {
   incident: 'incident'
 };
 
+export const MARKDOWN_ARTIFACTS_DEFAULT_SERVER_ADDRESS = 'http://localhost:10888';
```
Comment (dorsha) [inline src/constants/Constants.js:201]: What is this address?

File: src/components/Sections/SectionMarkdown.js [hunk 2]
```diff
@@ -76,7 +77,11 @@ export default class SectionMarkdown extends Component {
       }
       case 'img': {
         const srcArr = props.src.split('=size=');
-        props.src = srcArr[0];
+        if (srcArr[0].startsWith(MARKDOWN_IMAGES_PATH)) {
+          props.src = `${markdownArtifactsServerAddress}${srcArr[0]}`;
```
Comment (dorsha) [inline src/components/Sections/SectionMarkdown.js:81]: No need to add `/` if not exists?

File: src/constants/Constants.js (hunk 1 above)
Comment (davidbinyamin) [reply src/constants/Constants.js:201]: this is the default markdown artefact server that will be running in the same docker container.
you can see its implementation here: https://github.com/demisto/content/pull/14508

File: src/components/Sections/SectionMarkdown.js (hunk 2 above)
Comment (davidbinyamin) [reply src/components/Sections/SectionMarkdown.js:81]: the path in `MARKDOWN_IMAGES_PATH` already have `/`

File: src/constants/Constants.js (hunk 1 above)
Comment (dorsha) [reply src/constants/Constants.js:201]: It is too specific for the product, this is open source project, so better to provide it in the product itself.

Comment (davidbinyamin) [reply src/constants/Constants.js:201]: well... i do pass it as an argument from the script to reportService.
this is just a fallback default.

Comment (davidbinyamin) [reply src/constants/Constants.js:201]: i can remove the default.... and only rely on the argument received by the script

Comment (dorsha) [reply src/constants/Constants.js:201]: Let's remove.

**Review by dorsha — APPROVED** [review]

**Pull request merged**